src/utils/SpoonacularRequests.ts

```diff
@@ -18,6 +26,21 @@ export const fetchData = async(addRecipe: (title:string, id:number, image: strin
   const call = await fetch(`https://api.spoonacular.com/recipes/findByIngredients?apiKey=e0b11b97f0b64f5982330cabe8d0ec9b&ingredients=${ingredientSearch(ingredientList)}`);
 
   const response = await call.json();
+  console.log(response)
 
   onCallHandler(response, addRecipe, 5);
 }
+
+export const readRecipe = async(recipeId: number) => {
+  const url = `https://api.spoonacular.com/recipes/${recipeId.toString()}/information?apiKey=73a82c046a6043fd8fd23732ec9380c7&includeNutrition=false`
```

Comment:
lets use JSON stringify instead of toString

Reply:
but also I dont think you need either here, sincee its in a template string

Reply:
Changing this to be string since I will be grabbing the ID from the router params